CodeGeneration/Sources/SyntaxSupport/DeclNodes.swift

```diff
@@ -1179,6 +1179,9 @@ public let DECL_NODES: [Node] = [
           .keyword(text: "let"),
           .keyword(text: "func"),
           .keyword(text: "inout"),
+          .keyword(text: "mutating"),
+          .keyword(text: "consuming"),
+          .keyword(text: "borrowing"),
```

Comment:
Is `import mutating Module.var` something we want to support? Same for `consuming`. Could you also add test cases for it?

Reply:
@ahoppen are there any tests like this that exist already? I did a git grep for import var/let/etc.

Reply:
I don’t think so. DeclarationTests.swift seems like a good place to add them though